scripts/validate-autoinstall-user-data.py

```diff
@@ -27,64 +27,224 @@
 """
 
 import argparse
+import asyncio
 import io
-import json
+import sys
+import tempfile
+import traceback
+from argparse import Namespace
+from pathlib import Path
+from textwrap import dedent
+from typing import Any
 
-import jsonschema
 import yaml
 
+# Python path trickery so we can import subiquity code and still call this
+# script without using the makefile
+scripts_dir = sys.path[0]
+subiquity_root = Path(scripts_dir) / ".."
+curtin_root = subiquity_root / "curtin"
+probert_root = subiquity_root / "probert"
+# At the very least, local curtin needs to be in the front of the python path
+sys.path.insert(0, str(subiquity_root))
+sys.path.insert(1, str(curtin_root))
+sys.path.insert(2, str(probert_root))
+
+from subiquity.cmd.server import make_server_args_parser  # noqa: E402
+from subiquity.server.dryrun import DRConfig  # noqa: E402
+from subiquity.server.server import SubiquityServer  # noqa: E402
+
+
+SUCCESS_MSG = "Success: The provided autoinstall config validated succesfully"
+FAILURE_MSG = "Failure: The provided autoinstall config did not validate succesfully"
```

Comment:
nitpick: I'd reword just a bit so that the last words written are not "validate successfully". I'm a lazy reader :laughing: 

The provided autoinstall config failed validation?

Reply:
Also, I'm tempted to suggest `rich` (https://rich.readthedocs.io/en/stable/introduction.html) but it's probably difficult to justify for only one message.

Reply:
> lazy reader

Me too. I couldn't come up with something at the time but I like it. I used your suggested text. 

> I'm tempted to suggest `rich`

I do like the idea. And it's in Main, which is good. Although maybe once (if) we start having more error messaging we could use it?

Reply:
yeah, let's think about it later. This is not blocking.